packages/charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -84,58 +86,58 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
     axisNode.selectAll('text').attr('class', this._classNames.yAxisText);
   }
 
-  private _createNumericXAxis(isNumeric: boolean, data: IDataPoint[], width: number): numericAxis {
+  private _createNumericXAxis(isNumeric: boolean): numericAxis {
     let xMax;
     if (isNumeric) {
-      xMax = d3Max(data, (point: IDataPoint) => point.x as number)!;
+      xMax = d3Max(this._points, (point: IDataPoint) => point.x as number)!;
     } else {
-      xMax = d3Max(data, (point: IDataPoint) => point.y as number)!;
+      xMax = d3Max(this._points, (point: IDataPoint) => point.y as number)!;
     }
     const xAxisScale = d3ScaleLinear()
       .domain([0, xMax])
-      .range([0, width]);
+      .range([0, this._width]);
     const xAxis = d3AxisBottom(xAxisScale).ticks(10);
     return xAxis;
   }
 
-  private _createStringYAxis(data: IDataPoint[], height: number): stringAxis {
+  private _createStringYAxis(): stringAxis {
     const yAxisScale = d3ScaleBand()
-      .domain(data.map((point: IDataPoint) => point.x as string))
-      .range([height, 0]);
-    const yAxis = d3AxisLeft(yAxisScale).tickFormat((x: string, index: number) => data[index].x as string);
+      .domain(this._points.map((point: IDataPoint) => point.x as string))
+      .range([this._height, 0]);
+    const yAxis = d3AxisLeft(yAxisScale).tickFormat((x: string, index: number) => this._points[index].x as string);
     return yAxis;
   }
 
-  private _createYAxis(data: IDataPoint[], height: number, yAxisTickCount: number): numericAxis {
-    const yMax = d3Max(data, (point: IDataPoint) => point.y)!;
+  private _createYAxis(): numericAxis {
+    const yMax = d3Max(this._points, (point: IDataPoint) => point.y)!;
     const yAxisScale = d3ScaleLinear()
       .domain([0, yMax])
-      .range([height, 0]);
-    const yAxis = d3AxisLeft(yAxisScale).ticks(yAxisTickCount);
+      .range([this._height, 0]);
+    const yAxis = d3AxisLeft(yAxisScale).ticks(this._yAxisTickCount);
     return yAxis;
   }
 
-  private _createNumericBars(data: IDataPoint[], height: number, width: number, barHeight: number): JSX.Element[] {
```

Comment:
I generally like passing in arguments to utility functions (as stateless as possible) for unit testing. Do you think these functions should or will have unit tests?

Reply:
I wouldn't expect these to have unit tests since they're private methods.

Reply:
There are [ways around](https://medium.com/@macsikora/how-to-test-private-functions-of-es6-module-fb8c1345b25f) that and something I feel we should consider in general, particularly for such nice stateless functions. :)

Reply:
Always wondered about how to best test `private` functions.. thanks for sharing the above -- need to read tonight.

Reply:
Also, it's JavaScript. Nothing is really private. 😆

Reply:
I suspect you may just be able to do `(classInstance as any).privateFunction()` but I haven't tried it.